examples/kafka_spark_demo/feature_repo/streaming_features_tutorial.ipynb

```diff
@@ -0,0 +1,841 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Building streaming features with Feast, Spark, Kafka"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Overview\n",
+    "\n",
+    "This notebook builds off of the first [module](https://github.com/feast-dev/feast-workshop/tree/main/module_1) from the Feast workshop to explore how data scientists and engineers can build and register streaming features in Feast. Note that the workshop module is not a prerequisite for this tutorial, but is recommended if you are not already familiar with the Push API in Feast.\n",
+    "\n",
+    "Feast currently does not launch or manage any ingestion or stream transformation jobs. If you want to use streaming features, you will be responsible for ingesting data from your streams, applying any desired transformations, and then writing that data to the online store. Additionally, if you want to use that data for training models, you will also need to persist it to some batch data source, which must then be registered with Feast. (maybe should note unified push API here?)\n",
```

Comment:
do streaming features need to have batch sources? I could see for incremental adoption some users not wanting to use Feast for training at all and only wanting to manage the online store from a stream source.

Reply:
this is a fair point - I think this is non-blocking for a first cut, and if folks request this we can easily add it